Beta_Integrations/PaloAlto_TrapsESM_Beta/PaloAlto_TrapsESM_Beta.py

```diff
@@ -0,0 +1,268 @@
+        "Accept-Encoding": "gzip, deflate",
+        "Referer": URL + "/EndpointSecurityManager/HashManagement/Hashes",
+        "X-Requested-With": "XMLHttpRequest",
+        "Origin": URL,
+        "Content-Type": "application/json; charset=UTF-8",
+        "Token": token
+    }
+    payload_raw = {
+        'hash': file_hash
+    }
+
+    payload = json.dumps(payload_raw)
+
+    auth_cookie = {
+        'ct': cookies['ct'],
+        'csid': cookies['csid'],
+        'auth': cookies['auth']
+    }
+
+    result = requests.request('POST', URL + '/EndpointSecurityManager/HashManagement/HashesDetail',
+                              headers=headers, cookies=auth_cookie, data=payload, verify=USE_SSL)
+
+    hash_results = json.loads(result.content)
+
+    hr_table = {
+        'Result': hash_results['LocalAnalysis'][0]['Result'],
+        'Verdict History': hash_results['VerdictHistory'],
+        'Quarantined': hash_results['Quarantined']
+    }
+
+    ec = {
+        'TrapsESM': hash_results
+    }
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['markdown'],
+        'Contents': 'The verdict of hash {} is {}'.format(file_hash, hash_results['LocalAnalysis'][0]['Result']),
+        'HumanReadable': tableToMarkdown('Hash Verdict for {}'.format(file_hash), hr_table),
+        'EntryContext': ec
+    })
+
+
+def traps_esm_override_hash_verdict():
+    request_verification_token, cookies = get_auth_cookie()
+    token = get_new_request_token(cookies)
+    file_hash = demisto.args().get('hash')
+    verdict = demisto.args().get('verdict')
+    headers = {
+        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) "
+                      "Chrome/74.0.3729.131 Safari/537.36",
+        "Accept": "application/json, text/javascript, */*; q=0.01",
+        "Accept-Language": "Accept-Language:en-US,en;q=0.9",
+        "Accept-Encoding": "gzip, deflate",
+        "X-Requested-With": "XMLHttpRequest",
+        "Origin": URL,
+        "Content-Type": "application/json; charset=UTF-8",
+        "token": token
+    }
+    payload = {
+        'request': [file_hash],
+        'verdict': verdict
+    }
+
+    result = requests.request('POST', URL + '/EndpointSecurityManager/HashManagement/OverrideHashVerdict',
+                              headers=headers, cookies=cookies, data=payload, verify=USE_SSL)
+
+    demisto.results(result.content)
+    # res = result.json()
+    # if res['Succeed'] == True:
+    #     demisto.results('Hash verdict successfully updated.')
+    # else:
+    #     return_error('Hash verdict was not updated.')
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+LOG('Command being called is %s' % (demisto.command()))
+
+try:
+    if demisto.command() == 'test-module':
+        auth_cookie = get_auth_cookie()
+        if auth_cookie:
+            demisto.results('ok')
+        else:
+            demisto.results('definitely not okay')
```

Comment:
lets give something more descriptive here

Reply:
I'm removing the `else` so that a more descriptive error will be returned.